numbers_letters_and_variables/number_of_seconds.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+seconds_in_a_minute = 60
+minutes_in_a_hour = 60
+hours_in_a_day = 24
+days_in_a_week = 7
+days_in_a_year = 365
+age_twenty = (seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day * days_in_a_year * 20)
+age_hundred = (seconds_in_a_minute * minutes_in_a_hour * hours_in_a_day * days_in_a_year * 100)
+
+puts "There are #{seconds_in_a_minute} seconds in a minute"
+puts "There are #{minutes_in_a_hour} minutes in an hour"
+puts "There are #{hours_in_a_day} hours in a day"
+puts "There are #{days_in_a_week} days in a week"
+puts "That means there are:"
+puts "  #{seconds_in_a_minute * minutes_in_a_hour} seconds in an hour,"
```

Comment:
Could you pull out a `seconds_in_a_hour` variable above?  IMO, I like to have interpolation do as little calculation as possible...